builtin/logical/pki/cert_util.go

```diff
@@ -1340,6 +1341,24 @@ func generateCreationBundle(b *backend, data *inputBundle, caSign *certutil.CAIn
 		}
 	}
 
+	// Parse SKID from the request for cross-signing.
+	var skid []byte
+	{
+		if rawSKIDValue, ok := data.apiData.GetOk("skid"); ok {
+			var err error
+			skidValue := rawSKIDValue.(string)
+
+			// Handle removing common separators to make copy/paste from tool
+			// output easier.
+			skidNoColonsSeparators := strings.ReplaceAll(skidValue, ":", "")
+			skidNoSeparators := strings.ReplaceAll(skidNoColonsSeparators, "-", "")
```

Comment:
Could we add space here? (That's what's used in browser decoding)

Reply:
Ah interesting, Chrome uses a space but Firefox/OpenSSL use colons. Sure. :-)